Content.Server/Ghost/Roles/Components/GhostRoleTraitorReinforcementComponent.cs

```diff
@@ -0,0 +1,68 @@
+using Content.Server.Mind.Commands;
+using Content.Server.Mind.Components;
+using JetBrains.Annotations;
+using Robust.Server.Player;
+using Robust.Shared.Serialization.TypeSerializers.Implementations.Custom.Prototype;
+using Content.Server.Ghost.Roles.Events;
+using Content.Server.GameTicking.Rules;
+using Content.Server.Humanoid.Systems;
+using Content.Shared.Humanoid.Prototypes;
+
+namespace Content.Server.Ghost.Roles.Components
+{
+    [RegisterComponent, ComponentReference(typeof(GhostRoleComponent))]
+    public sealed class GhostRoleTraitorReinforcementComponent : GhostRoleComponent
```

Comment:
Please dont expand E/C code like this, i know GhostTakeoverAvailableComponent does this but that thing is incredibly old. If you want to do this, ECSing GhostRoleComponent is a requirement